nebuly-platform/templates/ingestion-worker_deployment.yaml

```
@@ -112,6 +117,15 @@ spec:
             {{- toYaml . | nindent 12 }}
           {{- end }}
       volumes:
+        - name: vllm-report-usage
+          emptyDir: {}
+        - name: models-cache
+      {{- if .Values.ingestionWorker.modelsCache.enabled }}
+          persistentVolumeClaim:
+            claimName: {{ include "ingestionWorker.modelsCache.name" . }}
+      {{- else }}
+          emptyDir: { }
+      {{- end }}
```

The models for PII removal are ~10MB, I think we can avoid the models cache disk here to avoid unnecessary complexity
```suggestion 
          emptyDir: { }
```

---

got it now. Fixed in https://github.com/nebuly-ai/helm-charts/pull/72/commits/5c916da7f2be19f9b4de2c84734ae2c162fcf97e